tests/integration/security/testdata/rbac/v1beta1-negative-match.yaml.tmpl

```diff
@@ -0,0 +1,128 @@
+# The following policy denies access to path with prefix "/prefix" except "/prefix/whitelist" to workload b
+
+apiVersion: "security.istio.io/v1beta1"
+kind: AuthorizationPolicy
+metadata:
+  name: policy-b-deny
+  namespace: "{{ .Namespace }}"
+spec:
+  selector:
+    matchLabels:
+      "app": "b"
+  action: DENY
+  rules:
+  - to:
+    - operation:
+        paths: ["/prefix*"]
+        notPaths: ["/prefix/whitelist"]
+---
+
+# The following policy denies access from other namespaces
+
+apiVersion: "security.istio.io/v1beta1"
+kind: AuthorizationPolicy
+metadata:
+  name: policy-c-same-namespace
+  namespace: "{{ .Namespace }}"
+spec:
+  selector:
+    matchLabels:
+      "app": "c"
+  action: DENY
+  rules:
+  - from:
+    - source:
+        notNamespaces: ["{{ .Namespace }}"]
+---
+
+# The following policy denies access to workload d if it's not mTLS, in other words,
+# it allows only mTLS traffic to access workload d
+
```

Comment:
As discussed, this example should be documented in official istio.io page.

Reply:
agree, I will send out the doc update to rigs.